### DISCUSSION TITLE
Return to the vanilla mob spawn radius

### DISCUSSION THREAD

**Discussion by Catana791** [Stardust-Labs-MC/StardustMC#30] [Ideas]
This has been discussed a lot recently so I want to suggest to go back to the vanilla mob spawn radius as a lot (most) farms are designed and built on these and become way slower or even stop working with other mob spawn radiuses.

**Comment by GinnyEvilShadow** [comment]
I second this

**Comment by aisukuma** [comment]
If you want the TPS lag to come back, then sure

**Reply by Catana791** [reply]
Hmm that's a very good point but it can't just be the mob spawn range that is causing tps issues can it?

**Reply by aisukuma** [reply]
Not just that but I’m playing around with settings, optimizing stuff is somewhat difficult

**Reply by Catana791** [reply]
I understand

**Comment by aisukuma** [comment]
Spawn radius reverted, I think. Let me know if the radius has been reverted after the upcoming restart
